bip-0068.mediawiki

```diff
@@ -30,7 +30,7 @@ If bit (1 << 31) of the sequence number is not set, then the sequence number is
 
 The sequence number encoding is interpreted as follows: 
 
-Bit (1 << 22) determines if the relative lock-time is time-based or block based: If the bit is set, the relative lock-time specifies a timespan in units of 512 seconds granularity. The timespan starts from the median-time-past (MTP) of the output’s previous block, and ends either at the MTP of the previous block or at the nTime of the transaction’s block (depending on the enforcement status of BIP113). If the bit is not set, the relative lock-time specifies a number of blocks.
+Bit (1 << 22) determines if the relative lock-time is time-based or block based: If the bit is set, the relative lock-time specifies a timespan in units of 512 seconds granularity. The timespan starts from the median-time-past (MTP) of the output’s previous block, and ends either at the MTP of the previous block or at the nTime of the transaction’s block as defined by BIP113. If the bit is not set, the relative lock-time specifies a number of blocks.
```

Comment:
I would just say this: ", and ends at the MTP of the previous block."   and get rid of the "or  ... BIP113"